### ISSUE TITLE
Rollup produces sourcemaps with sources as absolute paths

### ISSUE THREAD

**Issue by asapach** [systemjs/builder#705]
It looks like starting with rollup@0.34 the produced sourcemaps contain absolute paths instead of relative urls in `sources` when using input sourcemaps.

This looks something like this:
```js
{
  "version":3,
  "sources":["C:/tmp/rollup-sourcemaps/test.js"],
  "names":["foo"],
  "mappings":";;;AAAO,IAAMA,MAAM,KAAZ",
  "file":"out.js"
}
```
Repro case: https://github.com/asapach/rollup-sourcemaps-bug

Tracking this bug here for the time being.

**Comment by asapach** [comment]
Opened an issue with rollup: https://github.com/rollup/rollup/issues/1015

**Comment by guybedford** [comment]
Thanks @asapach, looks like a great reproduction there too. Happy to look at workarounds if it takes a while as well.

**Comment by asapach** [comment]
Since there seems to be no response from rollup, here's a workaround: #713

**Comment by guybedford** [comment]
Released in 0.15.33.
